app/lib/three_scale/oauth2/client.rb

```diff
@@ -72,7 +72,7 @@ def build_authentication(authentication_provider)
       # @return [::ThreeScale::OAuth::Client::Options]
       def build_authentication_options(authentication_provider)
         Options.new(
-          authentication_provider.site.presence,
+          authentication_provider.site.presence&.strip,
```

Comment:
Prevent server errors (see the PR description), for the `site` values that already exist in the DB and contain leading or trailing whitespaces.